lib/action_cable/channel/broadcasting.rb

```diff
@@ -0,0 +1,21 @@
+module ActionCable
+  module Channel
+    module Broadcasting
+      # Broadcast a hash a unique broadcasting for this <tt>object</tt> in this channel.
```

Comment:
I had missed a word, should have been "a hash to a unique broadcasting". I have been updated it.